### ISSUE TITLE
If the login form redirects to the same URL, nologinform is returned

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/autologin#11]
At least thats what is looks like:
```
DEBUG: Crawled (200) <GET http://localhost:8781> (referer: None) [<scrapy.http.cookies.CookieJar object at 0x10cfdbb10>]
INFO: found login form: {'fields': {'password': 'password', 'login': 'username'}, 'form': u'login'}
DEBUG: submit parameters: {'method': 'POST', 'headers': {'Content-Type': 'application/x-www-form-urlencoded'}, 'url': 'http://localhost:8781/', 'body': 'login=admin&password=secret'}
DEBUG: Redirecting (302) to <GET http://localhost:8781/> from <POST http://localhost:8781/>
DEBUG: Filtered duplicate request: <GET http://localhost:8781/> - no more duplicates will be shown (see DUPEFILTER_DEBUG to show all duplicates)
INFO: Closing spider (finished)
```

**Comment by kmike** [comment]
Yeah, it looks like the problem is that we don't take request headers in account in a dupefilter.

**Comment by lopuhin** [comment]
Fixed in 29f4413
